packages/bot-web-ui/src/components/dashboard/joyride-config.tsx

```diff
@@ -504,50 +504,62 @@ export const BOT_BUILDER_MOBILE: TStepMobile[] = [
         key: 6,
     },
 ];
-
 export const DBOT_ONBOARDING_MOBILE = [
+    {
+        header: localize('Get started on DBot'),
+        content: localize('Hi [first name]! Hit Start for a quick tour to help you get started.'),
```

Comment:
Is this the correct message we need to show?

Reply:
changed removed first name. Yes, we need to show this on the progress bar because we load the tour now on the first render and the tour is started now from the slider.